bin/guard

```diff
@@ -98,8 +94,8 @@ class GuardReloader
   end
 
   def auto_restart
-    args = [config.guard_core_path] + config.program_arguments
-    args.unshift(config.default_system_ruby) if config.windows?
+    args = [Gem.ruby, config.guard_core_path] + config.program_arguments
```

Comment:
With the change suggested above we would prepend `ruby` instead of `Gem.ruby`, i.e.

```ruby
args = [ruby, config.guard_core_path] + config.program_arguments
```

Reply:
I personally don't see much benefit in abstracting every method call. It makes reading code a little too hectic. 😇

Reply:
Yeah, we can use a local variable since it'd be called only once. ;)

Reply:
A local variable for what? :) `Gem.ruby` there as it is fits under 80 characters already. :P

Reply:
I meant for the `config.windows? ? RbConfig.ruby : Gem.ruby` proposal above, but I think `Gem.ruby` should work everywhere.